models/impossible_finance/bnb/impossible_finance_bnb_schema.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+
+models:
+  - name: impossible_finance_IDO_pools
+    meta:
+      blockchain: bnb
+      project: impossible_finance
+      contibutors: kartod
+    config:
+      tags: ['impossible', 'bnb']
+    description: "Impossible finance pools where users send BUSD to buy available allocation"
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain"
+
+      - &pool_address
```

Comment:
can we apply a unique test on just address?
if not only address, then use `unique_combo_of_columns` test